teuthology/orchestra/daemon/state.py

```diff
@@ -23,13 +23,13 @@ def __init__(self, remote, role, id_, *command_args, **command_kwargs):
         """
         self.remote = remote
         self.command_args = command_args
-        self.command_kwargs = command_kwargs
         self.role = role
         self.cluster, self.type_ = self.role.split('.')[0:2]
         self.id_ = id_
         self.log = command_kwargs.get('logger', log)
-        self.fsid = command_kwargs.get('fsid')
+        self.fsid = command_kwargs.pop('fsid')
```

Comment:
I guess you need `pop('fsid', None)` here?

Reply:
sigh, yes.. repushed